cpp/tests/reductions/scan_tests.cpp

```diff
@@ -535,26 +555,28 @@ TEST_F(StructScanTest, StructScanMinMaxWithNulls)
 
   {
     auto const expected = [] {
-      auto child1 = STRINGS_CW{"año",
-                               "año",
-                               "año",
-                               "" /*NULL*/,
-                               "" /*NULL*/,
-                               "" /*NULL*/,
-                               "" /*NULL*/,
-                               "" /*NULL*/,
-                               "" /*NULL*/,
-                               "" /*NULL*/};
-      auto child2 = INTS_CW{1,
-                            1,
-                            1,
-                            0 /*NULL*/,
-                            0 /*NULL*/,
-                            0 /*NULL*/,
-                            0 /*NULL*/,
-                            0 /*NULL*/,
-                            0 /*NULL*/,
-                            0 /*NULL*/};
+      auto child1 = STRINGS_CW{{"año",
+                                "año",
+                                "" /*NULL*/,
+                                "" /*NULL*/,
+                                "" /*NULL*/,
+                                "" /*NULL*/,
+                                "" /*NULL*/,
+                                "" /*NULL*/,
+                                "" /*NULL*/,
+                                "" /*NULL*/},
+                               null_at(2)};
```

Comment:
The `null_at(2)` does not seem to match the `NULL` comments above. Perhaps I'm reading this wrong.

Reply:
That child column has null at idx 2. The following nulls are at the parent column which are specified below (`nulls_at({3, 4, ...})`).